### PR TITLE
Ft create profile picture upload endpoint

### PR DESCRIPTION
**Pull request by Jobsy**
# Description

Describe change or new feature here.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: resources/auth/controller.js [hunk 1]
```diff
@@ -129,3 +129,17 @@ exports.viewProfile = async (req, res) => {
...
+
+exports.storeImgUrl = async (req, res) => {
+  try {
+    // const { id } = req;
+    // const { secure_url } = req.photos.info;
+    await model.updateImageUrl(6, 'yet another Test123');
+    res.status(200).json({
+      // id: id,
+      // imgUrl: secure_url,
+    });
```
Comment (emkayDauda) [inline resources/auth/controller.js]: Do you want to take these comments out, Joba?

**Review by Omulosi — CHANGES_REQUESTED** [review]
- I think you forgot to uncomment parts of your code that make up the logic of your functionality
- also, don't pass in a hardcoded ID value
otherwise, great job

File: resources/auth/controller.js (hunk 1 above)
Comment (emkayDauda) [reply resources/auth/controller.js]: Also looks like the id is hardcoded?

**Review by emkayDauda — CHANGES_REQUESTED** [review]

File: resources/auth/controller.js (hunk 1 above)
Comment (Jobsy) [reply resources/auth/controller.js]: Yeah!

Issue addressed

**Review by emkayDauda — DISMISSED** [review]
This looks good, @Jobsy, nicely done... 💪🏼

Would have been nice if we could do some schema validation  but it's just one variable so it's probably fine...

**Review by Omulosi — COMMENTED** [review]
Great work. Now you only need to add tests.

**Comment by Jobsy** [comment]
@emkayDauda schema added as requested
@Omulosi test also added as requested

Thank you guys for your feedbacks!

**Comment by emkayDauda** [comment]
@Jobsy, does this still require a review?

**Review by nobioma1 — CHANGES_REQUESTED** [review]
@Jobsy Good Job Joba, Looks Good 👍. So to foster same pattern throughout the code I'd suggest that your variable/function names start with a lowercase and uppercase for classes.

**Review by nobioma1 — APPROVED** [review]

**Pull request merged**